Add release note for 20201012 (#28)
tidbcloud/dbaas-docs: release-notes.md

This page lists the release notes of [TiDB Cloud](https://pingcap.com/products/tidbcloud).

## October 12, 2020

* Change SQL webshell client from Oracle MySQL client to usql

> * Upgrade default TiDB version to 4.0.7

Reviewer: ```suggestion
* Upgrade the default TiDB version to 4.0.7
```

> * Tune TiDB default configuration for t1.standard and h1.standard

Reviewer: ```suggestion
* Tune default TiDB configuration for t1.standard and h1.standard
```

> * Change manual backup reserve days from 7 to 30

Reviewer: Please check whether this is what you mean.
```suggestion
* Extend the manual backup retention period from 7 days to 30 days
```
